packages/web-components/fast-foundation/src/breadcrumb/README.md

```diff
@@ -0,0 +1,35 @@
+---
+id: breadcrumb
+title: fast-breadcrumb
+sidebar_label: breadcrumb
+custom_edit_url: https://github.com/microsoft/fast/edit/master/packages/web-components/fast-foundation/src/breadcrumb/README.md
+---
+
+A `fast-breadcrumb` widget is used as a navigational aid, allowing users to maintain awareness of their locations within a program, app, or a website.
```

Comment:
Could we avoid "widget"? Prefer "component" if necessary, but since the whole package is components we can often write the sentence without mentioning it.